README.md

```diff
@@ -42,15 +56,9 @@ To get a local copy up and running follow these simple example steps.
 
 👤 **Author1**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: [@ecekpo](https://github.com/ecekpo)
+- Twitter: [@ecekpo](https://twitter.com/ecekpo)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/ecekpo)
 
 ## 🤝 Contributing
 
```

Comment:
![mit-link](https://user-images.githubusercontent.com/93039359/186649251-37a1d19d-3ad4-4a1e-91c3-dfa214d5cd1c.png)

- [ ] Your link to MIT file is broken please fix it as shown below :+1:
`This project is [MIT](./MIT.md) licensed.`